build-system/tasks/size.js

```diff
@@ -33,6 +33,15 @@ var tableOptions = {
   hsep: '   |   ',
 };
 
+/**
+ * Through2 transform function - Tracks the original size and the gzipped size
+ * of the file content in the rows array. Passes the file and/or err to the
+ * callback when finished.
+ * @param {!Array<Array<string>>} rows array to store content size information
+ * @param {!File} file File to process
+ * @param {Object} enc Encoding (not used)
```

Comment:
honestly i'm not sure. but from what I can see it is always passed in and is probably a `string` actually, so that might be the more appropriate type.